packages/datatrak-web-server/src/models/SurveyResponse.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import {
+  SurveyResponseModel as BaseSurveyResponseModel,
+  SurveyResponseType as BaseSurveyResponseType,
+} from '@tupaia/database';
+import { Model } from '@tupaia/server-boilerplate';
+
+export type SurveyResponseModelFields = Readonly<{
+  id: string;
+  survey_id: string;
+  user_id: string;
+  start_time: string;
+  end_time: string;
+  metadata: Record<string, any>;
+  timezone: string | null;
+  entity_id: string;
+  data_time: string | null;
+  outdated: boolean | null;
+  approval_status: string;
+}>;
```

Comment:
The most up to date pattern here is to just use the type from `@tupaia/types` rather than defining all the fields here.

ie.
```
import {
  SurveyResponseModel as BaseSurveyResponseModel,
  SurveyResponseType as BaseSurveyResponseType,
} from '@tupaia/database';
import { SurveyResponse } from '@tupaia/types';

export interface SurveyResponseModelType
  extends SurveyResponse,
    Omit<BaseSurveyResponseType, 'id'> {} // Omit base `id: any` type as we explicity define as a string here

export interface SurveyResponseModel
  extends Model<BaseSurveyResponseModel, SurveyResponse, SurveyResponseModelType> {}
```

Reply:
Btw, I've got a [PR](https://github.com/beyondessential/tupaia/pull/5111) to centralise all these model definitions over to `@tupaia/server-boilerplate` so we don't need to keep defining them in the orchestration server. It just needs a bit of updating due to recent changes but I'll try and get it in soon :+1: